src/app/helpers/config/configHelper.js

```diff
@@ -0,0 +1,125 @@
+const envHelper = require('../environmentVariablesHelper.js')
+const configURL = envHelper.CONFIG_URL
+const apiAuthToken = envHelper.PORTAL_API_AUTH_TOKEN
+const _ = require('lodash')
+const request = require('request')
+const configModel = require('./configuration')
+const cron = require('node-cron');
+const configServiceEnabled = envHelper.CONFIG_SERVICE_ENABLED
+let configRefreshInterval = envHelper.CONFIG_REFRESH_INTERVAL
+let cacheRefreshEnabled = false
+const configReqKey = 'instance.portal'
+
+// left side 'key' is the configuration service key and right side 'value' is the envhelper key
+const configMap = {
+  sunbird_autocreate_trampoline_user: 'PORTAL_AUTOCREATE_TRAMPOLINE_USER',
+  sunbird_enabless_permission_check: 'ENABLE_PERMISSION_CHECK',
+  sunbird_instance: 'PORTAL_TITLE_NAME',
+  sunbird_theme: 'PORTAL_THEME',
+  sunbird_portal_default_language: 'PORTAL_DEFAULT_LANGUAGE',
```

Comment:
Is portal needed as it is not there in any other variable

Reply:
These are env variable names used in envvhelper and i followed the same.Can we use different names?